src/test/resources/sql/countryInsert.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS country (
```

Comment:
Next step is to get rid of manual table creation and deleting. Let hibernate deal with the table creation for testing.

Reply:
done